augur/application.py

```diff
@@ -4,92 +4,69 @@
 """
 
 import os
-import time
-import multiprocessing as mp
```

Comment:
I am assuming these imports are refactored elsewhere. Facade for sure needs multiprocessing.

Reply:
No, they're not. We just don't need these libraries for anything in the `Application()` class. This is entirely separate from Facade